saleor/graphql/webhook/subscription_types.py

```diff
@@ -1157,8 +1237,9 @@ def resolve_shipping_zone(root, _info: ResolveInfo, channel=None):
         return ChannelContext(node=shipping_method.shipping_zone, channel_slug=channel)
 
 
-class ShippingPriceCreated(ObjectType, ShippingPriceBase):
+class ShippingPriceCreated(SubscriptionObjectType, ShippingPriceBase):
     class Meta:
+        dry_run_model_name = "ShippingMethod"
```

Comment:
This one currently doesn't work. `get_node_or_error` function returns error instead of node, due to:
`graphene_type = info.schema.get_type("ShippingMethod).graphene_type`
`getattr(graphene_type, "get_node", None) = None`
Do you know how to handle it? I see that we have 2 options for `ShippingMethod` like `PRICE_BASED` and `WEIGHT_BASE`. Maybe it is somehow related...

Reply:
AFAIK ShippingMethod is a dynamic type, that can not be related to the DB model. The shipping method can come from the external apps. In the middle we're creating the dataclass that stores the object. Maybe this is something that causes the issue here 🤔

Reply:
Currently, ShippingPrice related events will be not supported